src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -747,6 +759,90 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
             break;
         }
 
+        // tell OS whether we're happy with the proposed changes to the window size
+        case WM_SIZING:
+        {
+            RECT* ncsRect = reinterpret_cast<RECT*>(lParam);
+            assert(ncsRect->left < ncsRect->right);
+            assert(ncsRect->top < ncsRect->bottom);
+            auto width  = static_cast<unsigned int>(ncsRect->right - ncsRect->left);
+            auto height = static_cast<unsigned int>(ncsRect->bottom - ncsRect->top);
+            bool isTop  = false; // are we being resized on the top? If not, it's the bottom
+            bool isLeft = false; // are we being resized on the left? If not, it's the right
+            switch (wParam)
+            {
+                case WMSZ_BOTTOMLEFT:
+                    isLeft = true;
+                    break;
+                case WMSZ_TOPLEFT:
+                    isLeft = true;
+                    isTop  = true;
+                    break;
+                case WMSZ_LEFT:
+                    isLeft = true;
+                    break;
+                case WMSZ_TOPRIGHT:
+                    isTop = true;
+                    break;
+                case WMSZ_TOP:
+                    isTop = true;
+                    break;
+            }
+
+
+            if (m_minimumSize && width < m_minimumSize->x)
+            {
+                // we don't like this
```

Comment:
These comments aren't obvious to me. Explain more about why we "don't like this".